src/components/results/detail/ImageDetail.vue

```diff
@@ -46,11 +46,12 @@
                       md="10"
                       offset-md="1"
                     >
-                      <div
-                        :class="{ blurExplicit: blurExplicit(file)}"
+                      <img
+                        :src="resourceURL"
                         class="image-wrapper"
-                        :style="{'background-image': `url(${resourceURL})`}"
-                      />
+                        :class="{ blurExplicit: blurExplicit(file)}"
+                      >
+
```

Comment:
would you mind removing this empty line still? Then it's perfect.

Reply:
sure!